network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -80,3 +80,12 @@ class PublicationPage(FoundationMetadataPageMixin, Page):
         FieldPanel('contents_title'),
         FieldPanel('notes')
     ]
+
+    def get_authors(self) -> set:
```

Comment:
I think the intention of this is to also check child pages for authors and percolate that up to the publication page, is that correct?

In which case, cool, but I'd like to double check that that is a requirement in spec. I got the impression from talks that some reports will define authors at the 'root' level, and some at the 'article' level, and so it would be up to editors in the CMS to define where they want author stuff to show. 

If not, this is a handy feature to have, and I'd want to make sure we communicate it as available functionality. *party emoji*

Reply:
I think this is a blocker. We need clarification before proceeding with this one.

Reply:
After design/dev sync today we confirmed: 

* Authors are optional and can be added to PublicationPage or ArticlePage
* Both pages will show Authors if they are selected for that page
* They will not show authors that have been selected for other pages